src/test/java/gov/usgs/wma/waterdata/collections/ObservationsStatTimeSeriesControllerIT.java

```diff
@@ -0,0 +1,84 @@
+package gov.usgs.wma.waterdata.collections;
+
+import static org.hamcrest.CoreMatchers.*;
+import static org.hamcrest.MatcherAssert.*;
+import static org.junit.jupiter.api.Assertions.*;
+import static uk.co.datumedge.hamcrest.json.SameJSONAs.*;
+import static gov.usgs.wma.waterdata.collections.ObservationsStatTimeSeriesController.*;
+
+import org.json.JSONObject;
+import org.junit.jupiter.api.Test;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.context.SpringBootTest.WebEnvironment;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+
+import com.github.springtestdbunit.annotation.DatabaseSetup;
+
+@SpringBootTest(webEnvironment=WebEnvironment.RANDOM_PORT)
+@DatabaseSetup("classpath:/testData/monitoringLocation/")
+@DatabaseSetup("classpath:/testData/groundwaterDailyValue/")
+public class ObservationsStatTimeSeriesControllerIT extends BaseCollectionsIT {
+	
+	
+	String makeURL(String collectionId, String featureId, String guid) {
```

Comment:
Maybe somewhat nitpicky: Prefer specific access declared if possible:  private, public, protected ...
if private, by convention we have been putting those methods after the public methods.

Reply:
It is a unit test and I am not a fussy as codecy for unit test, but it caught it too. Adding a descriptor. I personally think we should removed that option. Not for this case, but someone might want package accessor.

Reply:
It would be better IMO to have a local exception. I know you can do that in python with a pylint comment. I don't like turning off the rule because it more frequently finds the places where we should be using private.